tools/additional-per-crate-checks.sh

```diff
@@ -13,25 +13,21 @@ C_RESET='\033[0m'
 
 if [[ $# -lt 1 ]]; then
     echo "Usage: $0 [crate workspace path(s)...]"
-    echo "Visits all immediate sub-directories of the given workspace path,"
-    echo "looks for Cargo.toml files and additional-ci scripts. If both are"
-    echo "found, it executes the additional-ci script in that directory."
+    echo "Recursively visits 'additional-ci' scripts in the given directories and executes them."
```

Comment:
heh should we riir?

Reply:
Going to resist the urge this time around 😄